Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1407,6 +1406,8 @@ public static function resumeUserSession()
             !$GLOBALS['DIC']['ilAuthSession']->isAuthenticated() or
             $GLOBALS['DIC']['ilAuthSession']->isExpired()
         ) {
+            ilSession::_destroy($_COOKIE[session_name()],ilSession::SESSION_CLOSE_EXPIRE);
```

Comment:
Could we use `$DIC->http()->cookieJar()` instead (and for ILIAS >= 8.x `$DIC->http()->wrapper()->cookie()`, a PR for 8.x would be highly appreciated)?